websocket/websocket_test.go

```diff
@@ -1,18 +1,23 @@
 package websocket
 
 import (
+	"context"
 	"crypto/tls"
 	"crypto/x509"
-	"github.com/rs/zerolog"
+	"fmt"
 	"io"
 	"math/rand"
 	"net/http"
 	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
```

Comment:
Please keep 3rd party imports separate from project imports and sort. Your IDE or goimports can do this for you.